6bec05b25ff7926dd8a21a36ec059f3cc51b22ad: The C example in README does not work

This is what I got:

```
$ git clone git@github.com:libprima/prima.git && cd prima && git checkout  6bec05b25ff7926dd8a21a36ec059f3cc51b22ad
...

$ cmake -S . -B build -DCMAKE_INSTALL_PREFIX=install -DBUILD_SHARED_LIBS=OFF
cmake --build build --target install
-- Configuring done
-- Generating done
-- Build files have been written to: /home/zaikunzhang/tmp/prima/build
Scanning dependencies of target primaf
[  1%] Building Fortran object fortran/CMakeFiles/primaf.dir/common/linalg.f90.o
[  3%] Building Fortran object fortran/CMakeFiles/primaf.dir/common/pintrf.f90.o
[  4%] Building Fortran object fortran/CMakeFiles/primaf.dir/common/evaluate.f90.o
[  6%] Building Fortran object fortran/CMakeFiles/primaf.dir/common/powalg.f90.o
[  7%] Building Fortran object fortran/CMakeFiles/primaf.dir/bobyqa/geometry.f90.o
[  9%] Building Fortran object fortran/CMakeFiles/primaf.dir/common/history.f90.o
[ 10%] Building Fortran object fortran/CMakeFiles/primaf.dir/common/fprint.f90.o
[ 12%] Building Fortran object fortran/CMakeFiles/primaf.dir/common/message.f90.o
[ 14%] Building Fortran object fortran/CMakeFiles/primaf.dir/common/xinbd.f90.o
[ 15%] Building Fortran object fortran/CMakeFiles/primaf.dir/bobyqa/initialize.f90.o
[ 17%] Building Fortran object fortran/CMakeFiles/primaf.dir/common/ratio.f90.o
[ 18%] Building Fortran object fortran/CMakeFiles/primaf.dir/common/redrho.f90.o
[ 20%] Building Fortran object fortran/CMakeFiles/primaf.dir/bobyqa/rescue.f90.o
[ 21%] Building Fortran object fortran/CMakeFiles/primaf.dir/common/shiftbase.f90.o
[ 23%] Building Fortran object fortran/CMakeFiles/primaf.dir/common/univar.f90.o
[ 25%] Building Fortran object fortran/CMakeFiles/primaf.dir/bobyqa/trustregion.f90.o
[ 26%] Building Fortran object fortran/CMakeFiles/primaf.dir/bobyqa/update.f90.o
[ 28%] Building Fortran object fortran/CMakeFiles/primaf.dir/bobyqa/bobyqb.f90.o
[ 29%] Building Fortran object fortran/CMakeFiles/primaf.dir/common/preproc.f90.o
[ 31%] Building Fortran object fortran/CMakeFiles/primaf.dir/bobyqa/bobyqa.f90.o
[ 32%] Building Fortran object fortran/CMakeFiles/primaf.dir/cobyla/geometry.f90.o
[ 34%] Building Fortran object fortran/CMakeFiles/primaf.dir/common/selectx.f90.o
[ 35%] Building Fortran object fortran/CMakeFiles/primaf.dir/cobyla/initialize.f90.o
[ 37%] Building Fortran object fortran/CMakeFiles/primaf.dir/cobyla/trustregion.f90.o
[ 39%] Building Fortran object fortran/CMakeFiles/primaf.dir/cobyla/update.f90.o
[ 40%] Building Fortran object fortran/CMakeFiles/primaf.dir/cobyla/cobylb.f90.o
[ 42%] Building Fortran object fortran/CMakeFiles/primaf.dir/cobyla/cobyla.f90.o
[ 43%] Building Fortran object fortran/CMakeFiles/primaf.dir/lincoa/geometry.f90.o
[ 45%] Building Fortran object fortran/CMakeFiles/primaf.dir/lincoa/getact.f90.o
[ 46%] Building Fortran object fortran/CMakeFiles/primaf.dir/lincoa/initialize.f90.o
[ 48%] Building Fortran object fortran/CMakeFiles/primaf.dir/lincoa/trustregion.f90.o
[ 50%] Building Fortran object fortran/CMakeFiles/primaf.dir/lincoa/update.f90.o
[ 51%] Building Fortran object fortran/CMakeFiles/primaf.dir/lincoa/lincob.f90.o
[ 53%] Building Fortran object fortran/CMakeFiles/primaf.dir/lincoa/lincoa.f90.o
[ 54%] Building Fortran object fortran/CMakeFiles/primaf.dir/newuoa/geometry.f90.o
[ 56%] Building Fortran object fortran/CMakeFiles/primaf.dir/newuoa/initialize.f90.o
[ 57%] Building Fortran object fortran/CMakeFiles/primaf.dir/newuoa/trustregion.f90.o
[ 59%] Building Fortran object fortran/CMakeFiles/primaf.dir/newuoa/update.f90.o
[ 60%] Building Fortran object fortran/CMakeFiles/primaf.dir/newuoa/newuob.f90.o
[ 62%] Building Fortran object fortran/CMakeFiles/primaf.dir/newuoa/newuoa.f90.o
[ 64%] Building Fortran object fortran/CMakeFiles/primaf.dir/uobyqa/geometry.f90.o
[ 65%] Building Fortran object fortran/CMakeFiles/primaf.dir/uobyqa/initialize.f90.o
[ 67%] Building Fortran object fortran/CMakeFiles/primaf.dir/uobyqa/trustregion.f90.o
[ 68%] Building Fortran object fortran/CMakeFiles/primaf.dir/uobyqa/update.f90.o
[ 70%] Building Fortran object fortran/CMakeFiles/primaf.dir/uobyqa/uobyqb.f90.o
[ 71%] Building Fortran object fortran/CMakeFiles/primaf.dir/uobyqa/uobyqa.f90.o
[ 73%] Linking Fortran static library libprimaf.a
[ 87%] Built target primaf
Scanning dependencies of target primac
[ 89%] Building Fortran object c/CMakeFiles/primac.dir/cintrf.f90.o
[ 90%] Building Fortran object c/CMakeFiles/primac.dir/bobyqa_c.f90.o
[ 92%] Building Fortran object c/CMakeFiles/primac.dir/cobyla_c.f90.o
[ 93%] Building Fortran object c/CMakeFiles/primac.dir/lincoa_c.f90.o
[ 95%] Building Fortran object c/CMakeFiles/primac.dir/newuoa_c.f90.o
[ 96%] Building Fortran object c/CMakeFiles/primac.dir/uobyqa_c.f90.o
[ 98%] Building C object c/CMakeFiles/primac.dir/prima.c.o
[100%] Linking Fortran static library libprimac.a
[100%] Built target primac
Install the project...
-- Install configuration: "Release"
-- Installing: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a
-- Installing: /home/zaikunzhang/tmp/prima/install/lib/libprimac.a
-- Installing: /home/zaikunzhang/tmp/prima/install/lib/cmake/prima/prima-targets.cmake
-- Installing: /home/zaikunzhang/tmp/prima/install/lib/cmake/prima/prima-targets-release.cmake
-- Installing: /home/zaikunzhang/tmp/prima/install/lib/cmake/prima/prima-config.cmake
-- Installing: /home/zaikunzhang/tmp/prima/install/lib/cmake/prima/prima-config-version.cmake
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/bobyqb_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/huge_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/initialize_lincoa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/inf_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/uobyqb_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/consts_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/history_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/update_lincoa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/fprint_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/lincob_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/preproc_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/update_bobyqa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/trustregion_uobyqa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/debug_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/infos_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/redrho_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/update_newuoa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/cobyla_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/update_uobyqa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/geometry_lincoa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/trustregion_cobyla_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/univar_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/message_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/evaluate_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/newuob_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/xinbd_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/initialize_uobyqa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/string_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/selectx_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/bobyqa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/initialize_cobyla_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/geometry_bobyqa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/memory_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/cobylb_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/shiftbase_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/initialize_bobyqa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/linalg_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/geometry_newuoa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/update_cobyla_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/trustregion_newuoa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/uobyqa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/geometry_cobyla_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/newuoa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/infnan_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/trustregion_bobyqa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/trustregion_lincoa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/checkexit_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/ratio_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/pintrf_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/rescue_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/lincoa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/getact_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/initialize_newuoa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/geometry_uobyqa_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/mod/powalg_mod.mod
-- Installing: /home/zaikunzhang/tmp/prima/install/include/prima/prima.h
[26-Thu-22:34:00 zX11 ~/tmp/prima]
z$ cd fortran/examples/cobyla
cmake -S . -B build -DCMAKE_INSTALL_PREFIX=install -DPRIMA_DIR=$PWD/../../../install/lib/cmake/prima/
cmake --build build --target install
./install/bin/cobyla_example
-- The Fortran compiler identification is GNU 11.4.0
-- Detecting Fortran compiler ABI info
-- Detecting Fortran compiler ABI info - done
-- Check for working Fortran compiler: /usr/bin/f95 - skipped
-- Configuring done
-- Generating done
-- Build files have been written to: /home/zaikunzhang/tmp/prima/fortran/examples/cobyla/build
Scanning dependencies of target cobyla_example
[ 50%] Building Fortran object CMakeFiles/cobyla_example.dir/cobyla_example.f90.o
[100%] Linking Fortran executable cobyla_example
[100%] Built target cobyla_example
Install the project...
-- Install configuration: ""
-- Installing: /home/zaikunzhang/tmp/prima/fortran/examples/cobyla/install/bin/cobyla_example

Return from COBYLA because the trust region radius reaches its lower bound.
Number of function values = 642   Least value of F =  7.348005527767969E-011
The corresponding X is:
 6.687654693497622E-002   2.887373308458359E-001   3.666880613642483E-001   6.333153906725660E-001
 7.112679740025477E-001   9.331256571155258E-001

Return from COBYLA because the trust region radius reaches its lower bound.
Number of function values = 117   Least value of F = -8.660253768922790E-001   Constraint violation =  4.013687715520575E-010
The corresponding X is:
 8.661407979395734E-001   1.306240386801172E+000  -1.591365235013622E-010   9.999999998756539E-001
 8.660254038891088E-001   4.999999999312115E-001   2.308425905718184E-004   1.806440287472845E+000
 8.064403183272396E-001
The constraint value is:
-2.486921779620843E-010  -3.496540129762604E-001   1.125055049122636E-010  -9.698051695039567E-009
-3.499764252639898E-001  -8.595537948075105E-009   4.013687715520575E-010  -3.496540092517841E-001
-8.420485864135685E-009  -8.661407980397428E-001   1.283341086699288E-010  -6.983978023918092E-001
-1.564307758264958E+000  -8.064403183272396E-001
[26-Thu-22:34:05 zX11 ~/tmp/prima/fortran/examples/cobyla]
z$ cd ../../../
[26-Thu-22:34:15 zX11 ~/tmp/prima]
z$ cd c/examples/cobyla
cmake -S . -B build -DCMAKE_INSTALL_PREFIX=install -DPRIMA_DIR=$PWD/../../../install/lib/cmake/prima/
cmake --build build --target install
./install/bin/cobyla_example
-- The C compiler identification is GNU 11.4.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Configuring done
-- Generating done
-- Build files have been written to: /home/zaikunzhang/tmp/prima/c/examples/cobyla/build
[ 50%] Building C object CMakeFiles/cobyla_example.dir/cobyla_example.c.o
[100%] Linking C executable cobyla_example
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(cobyla.f90.o): in function `__cobyla_mod_MOD_cobyla':
cobyla.f90:(.text+0x2d96): undefined reference to `_gfortran_shape_4'
/usr/bin/ld: cobyla.f90:(.text+0x2e35): undefined reference to `_gfortran_reshape_r8'
/usr/bin/ld: cobyla.f90:(.text+0x4948): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: cobyla.f90:(.text+0x498e): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: cobyla.f90:(.text+0x4f73): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x4fa1): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x4fc9): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x61fd): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x625b): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x6a68): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x6a80): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x6a9c): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x6ab4): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x6ad0): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x6aec): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x6b08): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x6b20): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x6b3c): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x6b52): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x6b6a): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x6b86): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x6ba2): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x6bbe): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x6bda): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x6bf2): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x6c0e): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: cobyla.f90:(.text+0x6c2a): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(cobylb.f90.o): in function `__cobylb_mod_MOD_cobylb':
cobylb.f90:(.text+0x17a4): undefined reference to `_gfortran_minval_r8'
/usr/bin/ld: cobylb.f90:(.text+0x1ab7): undefined reference to `_gfortran_maxval_r8'
/usr/bin/ld: cobylb.f90:(.text+0x2a90): undefined reference to `_gfortran_spread'
/usr/bin/ld: cobylb.f90:(.text+0x3b4f): undefined reference to `_gfortran_spread'
/usr/bin/ld: cobylb.f90:(.text+0x489c): undefined reference to `_gfortran_minval_r8'
/usr/bin/ld: cobylb.f90:(.text+0x4d40): undefined reference to `_gfortran_maxval_r8'
/usr/bin/ld: cobylb.f90:(.text+0x9218): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: cobylb.f90:(.text+0x927d): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: cobylb.f90:(.text+0x929a): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: cobylb.f90:(.text+0x92f7): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: cobylb.f90:(.text+0x93e2): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(geometry.f90.o): in function `__geometry_cobyla_mod_MOD_geostep':
geometry.f90:(.text+0x72a): undefined reference to `_gfortran_spread'
/usr/bin/ld: geometry.f90:(.text+0x1a41): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: geometry.f90:(.text+0x1a59): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(geometry.f90.o): in function `__geometry_cobyla_mod_MOD_setdrop_tr':
geometry.f90:(.text+0x22c8): undefined reference to `_gfortran_spread'
/usr/bin/ld: geometry.f90:(.text+0x2b43): undefined reference to `_gfortran_spread'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(initialize.f90.o): in function `__initialize_cobyla_mod_MOD_initxfc':
initialize.f90:(.text+0x2fba): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(trustregion.f90.o): in function `__trustregion_cobyla_mod_MOD_trstlp_sub.constprop.0':
trustregion.f90:(.text+0x2f9a): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: trustregion.f90:(.text+0x4f8d): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: trustregion.f90:(.text+0x504b): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: trustregion.f90:(.text+0x5241): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(trustregion.f90.o):trustregion.f90:(.text+0x525e): more undefined references to `_gfortran_os_error_at' follow
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(trustregion.f90.o): in function `__trustregion_cobyla_mod_MOD_trstlp':
trustregion.f90:(.text+0x5875): undefined reference to `_gfortran_reshape_r8'
/usr/bin/ld: trustregion.f90:(.text+0x6283): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(update.f90.o): in function `__update_cobyla_mod_MOD_updatepole':
update.f90:(.text+0xc05): undefined reference to `_gfortran_spread'
/usr/bin/ld: update.f90:(.text+0x1daf): undefined reference to `_gfortran_spread'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(update.f90.o): in function `__update_cobyla_mod_MOD_updatexfc':
update.f90:(.text+0x3ee7): undefined reference to `_gfortran_spread'
/usr/bin/ld: update.f90:(.text+0x4c12): undefined reference to `_gfortran_spread'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(bobyqa.f90.o): in function `__bobyqa_mod_MOD_bobyqa':
bobyqa.f90:(.text+0x840): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: bobyqa.f90:(.text+0x886): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: bobyqa.f90:(.text+0x1b4e): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: bobyqa.f90:(.text+0x1be8): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: bobyqa.f90:(.text+0x1c2e): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: bobyqa.f90:(.text+0x1e48): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(bobyqb.f90.o): in function `__bobyqb_mod_MOD_bobyqb':
bobyqb.f90:(.text+0x2c54): undefined reference to `_gfortran_spread'
/usr/bin/ld: bobyqb.f90:(.text+0x4e7e): undefined reference to `_gfortran_spread'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(geometry.f90.o): in function `__geometry_bobyqa_mod_MOD_geostep':
geometry.f90:(.text+0x118f): undefined reference to `_gfortran_spread'
/usr/bin/ld: geometry.f90:(.text+0x1f20): undefined reference to `lround'
/usr/bin/ld: geometry.f90:(.text+0x2570): undefined reference to `lround'
/usr/bin/ld: geometry.f90:(.text+0x273b): undefined reference to `_gfortran_spread'
/usr/bin/ld: geometry.f90:(.text+0x2a5f): undefined reference to `_gfortran_spread'
/usr/bin/ld: geometry.f90:(.text+0x2e7e): undefined reference to `_gfortran_maxloc1_4_r8'
/usr/bin/ld: geometry.f90:(.text+0x417b): undefined reference to `_gfortran_spread'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(geometry.f90.o): in function `__geometry_bobyqa_mod_MOD_setdrop_tr':
geometry.f90:(.text+0x76b7): undefined reference to `_gfortran_spread'
/usr/bin/ld: geometry.f90:(.text+0x7e7f): undefined reference to `_gfortran_spread'
/usr/bin/ld: geometry.f90:(.text+0x8050): undefined reference to `_gfortran_spread'
/usr/bin/ld: geometry.f90:(.text+0x8210): undefined reference to `_gfortran_spread'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(trustregion.f90.o): in function `__trustregion_bobyqa_mod_MOD_trsbox':
trustregion.f90:(.text+0x2a6f): undefined reference to `lround'
/usr/bin/ld: trustregion.f90:(.text+0x6bd9): undefined reference to `lround'
/usr/bin/ld: trustregion.f90:(.text+0x74e9): undefined reference to `lround'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(rescue.f90.o): in function `__rescue_mod_MOD_rescue':
rescue.f90:(.text+0xc0a): undefined reference to `_gfortran_spread'
/usr/bin/ld: rescue.f90:(.text+0x2807): undefined reference to `_gfortran_spread'
/usr/bin/ld: rescue.f90:(.text+0x2938): undefined reference to `log10f'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(lincoa.f90.o): in function `__lincoa_mod_MOD_lincoa':
lincoa.f90:(.text+0x354b): undefined reference to `_gfortran_shape_4'
/usr/bin/ld: lincoa.f90:(.text+0x35e7): undefined reference to `_gfortran_reshape_r8'
/usr/bin/ld: lincoa.f90:(.text+0x43da): undefined reference to `_gfortran_spread'
/usr/bin/ld: lincoa.f90:(.text+0x53b5): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: lincoa.f90:(.text+0x583d): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: lincoa.f90:(.text+0x5883): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: lincoa.f90:(.text+0x67aa): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: lincoa.f90:(.text+0x7017): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: lincoa.f90:(.text+0x7033): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: lincoa.f90:(.text+0x704f): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: lincoa.f90:(.text+0x706b): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: lincoa.f90:(.text+0x7087): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(lincoa.f90.o):lincoa.f90:(.text+0x70a3): more undefined references to `_gfortran_runtime_error_at' follow
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(lincoa.f90.o): in function `__lincoa_mod_MOD_lincoa':
lincoa.f90:(.text+0x70f3): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: lincoa.f90:(.text+0x710b): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: lincoa.f90:(.text+0x7123): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: lincoa.f90:(.text+0x713f): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: lincoa.f90:(.text+0x7157): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: lincoa.f90:(.text+0x7173): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: lincoa.f90:(.text+0x718b): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: lincoa.f90:(.text+0x71a3): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: lincoa.f90:(.text+0x71bb): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: lincoa.f90:(.text+0x71d7): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: lincoa.f90:(.text+0x71f3): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: lincoa.f90:(.text+0x720f): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: lincoa.f90:(.text+0x722b): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: lincoa.f90:(.text+0x7247): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(newuoa.f90.o):newuoa.f90:(.text+0xcef): more undefined references to `_gfortran_runtime_error_at' follow
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(newuoa.f90.o): in function `__newuoa_mod_MOD_newuoa':
newuoa.f90:(.text+0xde7): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: newuoa.f90:(.text+0xe2d): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: newuoa.f90:(.text+0xfce): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(uobyqa.f90.o): in function `__uobyqa_mod_MOD_uobyqa':
uobyqa.f90:(.text+0xcca): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: uobyqa.f90:(.text+0xdc7): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: uobyqa.f90:(.text+0xe0d): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: uobyqa.f90:(.text+0xfae): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(linalg.f90.o): in function `__linalg_mod_MOD_vec2smat':
linalg.f90:(.text+0x6a7): undefined reference to `lroundf'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(linalg.f90.o): in function `__linalg_mod_MOD_linspace_i':
linalg.f90:(.text+0x109b): undefined reference to `lround'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(linalg.f90.o): in function `__linalg_mod_MOD_trueloc':
linalg.f90:(.text+0x1d59): undefined reference to `_gfortran_pack'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(linalg.f90.o): in function `__linalg_mod_MOD_sort_i1':
linalg.f90:(.text+0x317a): undefined reference to `_gfortran_compare_string'
/usr/bin/ld: linalg.f90:(.text+0x328a): undefined reference to `_gfortran_compare_string'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(linalg.f90.o): in function `__linalg_mod_MOD_sort_i2':
linalg.f90:(.text+0x34b1): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: linalg.f90:(.text+0x34cf): undefined reference to `_gfortran_string_trim'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(linalg.f90.o): in function `__linalg_mod_MOD_named_norm_mat':
linalg.f90:(.text+0x401b): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: linalg.f90:(.text+0x4043): undefined reference to `_gfortran_string_trim'
/usr/bin/ld: linalg.f90:(.text+0x4152): undefined reference to `_gfortran_select_string'
/usr/bin/ld: linalg.f90:(.text+0x41aa): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: linalg.f90:(.text+0x41cf): undefined reference to `_gfortran_string_trim'
/usr/bin/ld: linalg.f90:(.text+0x4246): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: linalg.f90:(.text+0x42c2): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: linalg.f90:(.text+0x4728): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(linalg.f90.o): in function `__linalg_mod_MOD_p_norm':
linalg.f90:(.text+0x4aa7): undefined reference to `pow'
/usr/bin/ld: linalg.f90:(.text+0x4e0c): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(linalg.f90.o): in function `__linalg_mod_MOD_named_norm_vec':
linalg.f90:(.text+0x4fce): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: linalg.f90:(.text+0x4ff6): undefined reference to `_gfortran_string_trim'
/usr/bin/ld: linalg.f90:(.text+0x50d3): undefined reference to `_gfortran_select_string'
/usr/bin/ld: linalg.f90:(.text+0x52b4): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: linalg.f90:(.text+0x52d4): undefined reference to `_gfortran_string_trim'
/usr/bin/ld: linalg.f90:(.text+0x5343): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: linalg.f90:(.text+0x53ad): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: linalg.f90:(.text+0x5563): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(linalg.f90.o): in function `__linalg_mod_MOD_planerot':
linalg.f90:(.text+0x5f6b): undefined reference to `_gfortran_reshape_r8'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(linalg.f90.o): in function `__linalg_mod_MOD_diag':
linalg.f90:(.text+0x6c40): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: linalg.f90:(.text+0x6c55): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(linalg.f90.o): in function `__linalg_mod_MOD_lsqr_rdiag':
linalg.f90:(.text+0xeee3): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(linalg.f90.o): in function `__linalg_mod_MOD_p_norm':
linalg.f90:(.text+0x4af4): undefined reference to `pow'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(string.f90.o): in function `__string_mod_MOD_strip':
string.f90:(.text+0x2e9): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: string.f90:(.text+0x2fc): undefined reference to `_gfortran_string_trim'
/usr/bin/ld: string.f90:(.text+0x330): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(string.f90.o): in function `__string_mod_MOD_int2str':
string.f90:(.text+0x40d): undefined reference to `_gfortran_st_write'
/usr/bin/ld: string.f90:(.text+0x41d): undefined reference to `_gfortran_transfer_integer_write'
/usr/bin/ld: string.f90:(.text+0x42b): undefined reference to `_gfortran_st_write_done'
/usr/bin/ld: string.f90:(.text+0x43b): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: string.f90:(.text+0x452): undefined reference to `_gfortran_string_trim'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(string.f90.o): in function `__string_mod_MOD_real2str_scalar':
string.f90:(.text+0x610): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: string.f90:(.text+0x6bc): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: string.f90:(.text+0x713): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: string.f90:(.text+0x7a0): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: string.f90:(.text+0x809): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(string.f90.o):string.f90:(.text+0x8a4): more undefined references to `_gfortran_concat_string' follow
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(string.f90.o): in function `__string_mod_MOD_real2str_scalar':
string.f90:(.text+0x9c0): undefined reference to `_gfortran_st_write'
/usr/bin/ld: string.f90:(.text+0x9d0): undefined reference to `_gfortran_transfer_real_write'
/usr/bin/ld: string.f90:(.text+0x9d8): undefined reference to `_gfortran_st_write_done'
/usr/bin/ld: string.f90:(.text+0x9e5): undefined reference to `_gfortran_string_len_trim'
/usr/bin/ld: string.f90:(.text+0xad2): undefined reference to `_gfortran_st_write'
/usr/bin/ld: string.f90:(.text+0xae2): undefined reference to `_gfortran_transfer_real_write'
/usr/bin/ld: string.f90:(.text+0xaea): undefined reference to `_gfortran_st_write_done'
/usr/bin/ld: string.f90:(.text+0xafa): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: string.f90:(.text+0xb17): undefined reference to `_gfortran_string_trim'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(preproc.f90.o): in function `__preproc_mod_MOD_preproc':
preproc.f90:(.text+0x2bb): undefined reference to `_gfortran_compare_string'
/usr/bin/ld: preproc.f90:(.text+0x345): undefined reference to `_gfortran_select_string'
/usr/bin/ld: preproc.f90:(.text+0x432): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: preproc.f90:(.text+0x460): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: preproc.f90:(.text+0x4ed): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: preproc.f90:(.text+0x5e8): undefined reference to `_gfortran_compare_string'
/usr/bin/ld: preproc.f90:(.text+0x60e): undefined reference to `_gfortran_compare_string'
/usr/bin/ld: preproc.f90:(.text+0x70f): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: preproc.f90:(.text+0x799): undefined reference to `_gfortran_compare_string'
/usr/bin/ld: preproc.f90:(.text+0x7b8): undefined reference to `_gfortran_compare_string'
/usr/bin/ld: preproc.f90:(.text+0x8b0): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: preproc.f90:(.text+0x9e1): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: preproc.f90:(.text+0xb13): undefined reference to `_gfortran_compare_string'
/usr/bin/ld: preproc.f90:(.text+0xd05): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: preproc.f90:(.text+0xd53): undefined reference to `_gfortran_compare_string'
/usr/bin/ld: preproc.f90:(.text+0xe15): undefined reference to `_gfortran_select_string'
/usr/bin/ld: preproc.f90:(.text+0xf32): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: preproc.f90:(.text+0xf82): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: preproc.f90:(.text+0x1065): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: preproc.f90:(.text+0x10ec): undefined reference to `_gfortran_compare_string'
/usr/bin/ld: preproc.f90:(.text+0x130a): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: preproc.f90:(.text+0x1341): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: preproc.f90:(.text+0x1385): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: preproc.f90:(.text+0x1451): undefined reference to `_gfortran_compare_string'
/usr/bin/ld: preproc.f90:(.text+0x14eb): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: preproc.f90:(.text+0x15e5): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: preproc.f90:(.text+0x2203): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: preproc.f90:(.text+0x2249): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: preproc.f90:(.text+0x2292): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(preproc.f90.o):preproc.f90:(.text+0x2485): more undefined references to `_gfortran_concat_string' follow
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(preproc.f90.o): in function `__preproc_mod_MOD_preproc':
preproc.f90:(.text+0x3a33): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: preproc.f90:(.text+0x3a4b): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(powalg.f90.o): in function `__powalg_mod_MOD_errh':
powalg.f90:(.text+0x2902): undefined reference to `_gfortran_maxval_r8'
/usr/bin/ld: powalg.f90:(.text+0x2a09): undefined reference to `_gfortran_minval_r8'
/usr/bin/ld: powalg.f90:(.text+0x2c02): undefined reference to `_gfortran_maxval_r8'
/usr/bin/ld: powalg.f90:(.text+0x2d1d): undefined reference to `_gfortran_minval_r8'
/usr/bin/ld: powalg.f90:(.text+0x3abd): undefined reference to `_gfortran_minval_r8'
/usr/bin/ld: powalg.f90:(.text+0x3cfa): undefined reference to `_gfortran_minval_r8'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(powalg.f90.o): in function `__powalg_mod_MOD_calvlag_lfqint':
powalg.f90:(.text+0x6d08): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: powalg.f90:(.text+0x6d20): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(powalg.f90.o): in function `__powalg_mod_MOD_errquad':
powalg.f90:(.text+0xcfe3): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(history.f90.o): in function `__history_mod_MOD_rangehist':
history.f90:(.text+0xa31): undefined reference to `_gfortran_shape_4'
/usr/bin/ld: history.f90:(.text+0xac6): undefined reference to `_gfortran_reshape_r8'
/usr/bin/ld: history.f90:(.text+0xeff): undefined reference to `_gfortran_shape_4'
/usr/bin/ld: history.f90:(.text+0xf94): undefined reference to `_gfortran_reshape_r8'
/usr/bin/ld: history.f90:(.text+0x1401): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: history.f90:(.text+0x1419): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: history.f90:(.text+0x1431): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(message.f90.o): in function `__message_mod_MOD_fmsg':
message.f90:(.text+0x17f): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0x1b2): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0x237): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0x2ff): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0x345): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(message.f90.o):message.f90:(.text+0x38b): more undefined references to `_gfortran_concat_string' follow
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(message.f90.o): in function `__message_mod_MOD_fmsg':
message.f90:(.text+0xa47): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: message.f90:(.text+0xa76): undefined reference to `_gfortran_string_trim'
/usr/bin/ld: message.f90:(.text+0xaee): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0xbd3): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0xc19): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0xc5f): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0xce9): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(message.f90.o):message.f90:(.text+0xd8c): more undefined references to `_gfortran_concat_string' follow
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(message.f90.o): in function `__message_mod_MOD_fmsg':
message.f90:(.text+0x10bd): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(message.f90.o): in function `__message_mod_MOD_cpenmsg':
message.f90:(.text+0x1196): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0x1263): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: message.f90:(.text+0x1280): undefined reference to `_gfortran_string_trim'
/usr/bin/ld: message.f90:(.text+0x12f6): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0x13f6): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(message.f90.o): in function `__message_mod_MOD_rhomsg':
message.f90:(.text+0x15f2): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0x1642): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0x1692): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(message.f90.o):message.f90:(.text+0x172d): more undefined references to `_gfortran_concat_string' follow
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(message.f90.o): in function `__message_mod_MOD_rhomsg':
message.f90:(.text+0x2107): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: message.f90:(.text+0x2136): undefined reference to `_gfortran_string_trim'
/usr/bin/ld: message.f90:(.text+0x21ae): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0x228d): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0x22d3): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0x2319): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0x239f): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(message.f90.o):message.f90:(.text+0x2440): more undefined references to `_gfortran_concat_string' follow
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(message.f90.o): in function `__message_mod_MOD_rhomsg':
message.f90:(.text+0x2a4d): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(message.f90.o): in function `__message_mod_MOD_retmsg':
message.f90:(.text+0x2c10): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: message.f90:(.text+0x2c35): undefined reference to `_gfortran_string_trim'
/usr/bin/ld: message.f90:(.text+0x2ca9): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0x2da2): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0x2dd5): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0x2e05): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: message.f90:(.text+0x2e2d): undefined reference to `_gfortran_string_trim'
/usr/bin/ld: message.f90:(.text+0x2ea6): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0x3012): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0x30de): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0x3124): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: message.f90:(.text+0x316a): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(message.f90.o):message.f90:(.text+0x31f4): more undefined references to `_gfortran_concat_string' follow
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(message.f90.o): in function `__message_mod_MOD_retmsg':
message.f90:(.text+0x410d): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(fprint.f90.o): in function `__fprint_mod_MOD_fprint':
fprint.f90:(.text+0xdb): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: fprint.f90:(.text+0x16d): undefined reference to `_gfortran_select_string'
/usr/bin/ld: fprint.f90:(.text+0x1f8): undefined reference to `_gfortran_st_inquire'
/usr/bin/ld: fprint.f90:(.text+0x2d7): undefined reference to `_gfortran_st_open'
/usr/bin/ld: fprint.f90:(.text+0x311): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: fprint.f90:(.text+0x3c8): undefined reference to `_gfortran_string_index'
/usr/bin/ld: fprint.f90:(.text+0x449): undefined reference to `_gfortran_st_write'
/usr/bin/ld: fprint.f90:(.text+0x46e): undefined reference to `_gfortran_transfer_character_write'
/usr/bin/ld: fprint.f90:(.text+0x476): undefined reference to `_gfortran_st_write_done'
/usr/bin/ld: fprint.f90:(.text+0x4b8): undefined reference to `_gfortran_string_index'
/usr/bin/ld: fprint.f90:(.text+0x4db): undefined reference to `_gfortran_string_len_trim'
/usr/bin/ld: fprint.f90:(.text+0x51a): undefined reference to `_gfortran_st_close'
/usr/bin/ld: fprint.f90:(.text+0x5bf): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: fprint.f90:(.text+0x5ff): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: fprint.f90:(.text+0x698): undefined reference to `_gfortran_st_write'
/usr/bin/ld: fprint.f90:(.text+0x6a6): undefined reference to `_gfortran_transfer_character_write'
/usr/bin/ld: fprint.f90:(.text+0x6ae): undefined reference to `_gfortran_st_write_done'
/usr/bin/ld: fprint.f90:(.text+0x7ca): undefined reference to `_gfortran_st_open'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(debug.F90.o): in function `__debug_mod_MOD_warning':
debug.F90:(.text+0x63): undefined reference to `_gfortran_st_write'
/usr/bin/ld: debug.F90:(.text+0x85): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: debug.F90:(.text+0x9a): undefined reference to `_gfortran_string_trim'
/usr/bin/ld: debug.F90:(.text+0xdc): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: debug.F90:(.text+0x128): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: debug.F90:(.text+0x154): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: debug.F90:(.text+0x169): undefined reference to `_gfortran_string_trim'
/usr/bin/ld: debug.F90:(.text+0x1ae): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: debug.F90:(.text+0x20d): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: debug.F90:(.text+0x223): undefined reference to `_gfortran_transfer_character_write'
/usr/bin/ld: debug.F90:(.text+0x233): undefined reference to `_gfortran_st_write_done'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(debug.F90.o): in function `__debug_mod_MOD_errstop':
debug.F90:(.text+0x2c8): undefined reference to `_gfortran_st_write'
/usr/bin/ld: debug.F90:(.text+0x2ea): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: debug.F90:(.text+0x2ff): undefined reference to `_gfortran_string_trim'
/usr/bin/ld: debug.F90:(.text+0x341): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: debug.F90:(.text+0x38b): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: debug.F90:(.text+0x3b7): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: debug.F90:(.text+0x3cc): undefined reference to `_gfortran_string_trim'
/usr/bin/ld: debug.F90:(.text+0x40e): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: debug.F90:(.text+0x468): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: debug.F90:(.text+0x47e): undefined reference to `_gfortran_transfer_character_write'
/usr/bin/ld: debug.F90:(.text+0x48e): undefined reference to `_gfortran_st_write_done'
/usr/bin/ld: debug.F90:(.text+0x4a1): undefined reference to `_gfortran_error_stop_numeric'
/usr/bin/ld: debug.F90:(.text+0x4af): undefined reference to `_gfortran_error_stop_string'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(debug.F90.o): in function `__debug_mod_MOD_validate':
debug.F90:(.text+0x518): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: debug.F90:(.text+0x52b): undefined reference to `_gfortran_string_trim'
/usr/bin/ld: debug.F90:(.text+0x54d): undefined reference to `_gfortran_adjustl'
/usr/bin/ld: debug.F90:(.text+0x562): undefined reference to `_gfortran_string_trim'
/usr/bin/ld: debug.F90:(.text+0x5a1): undefined reference to `_gfortran_concat_string'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(memory.F90.o): in function `__memory_mod_MOD_alloc_character':
memory.F90:(.text+0x201): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(selectx.f90.o): in function `__selectx_mod_MOD_isbetter01':
selectx.f90:(.text+0x488): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(selectx.f90.o): in function `__selectx_mod_MOD_isbetter10':
selectx.f90:(.text+0x908): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(shiftbase.f90.o): in function `__shiftbase_mod_MOD_shiftbase_qint':
shiftbase.f90:(.text+0x3a7): undefined reference to `_gfortran_spread'
/usr/bin/ld: shiftbase.f90:(.text+0xa5e): undefined reference to `_gfortran_spread'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(shiftbase.f90.o): in function `__shiftbase_mod_MOD_shiftbase_lfqint':
shiftbase.f90:(.text+0x12ea): undefined reference to `_gfortran_spread'
/usr/bin/ld: shiftbase.f90:(.text+0x2ce2): undefined reference to `_gfortran_spread'
/usr/bin/ld: shiftbase.f90:(.text+0x31f2): undefined reference to `_gfortran_spread'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(lincob.f90.o):lincob.f90:(.text+0x518d): more undefined references to `_gfortran_spread' follow
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(lincob.f90.o): in function `__lincob_mod_MOD_lincob':
lincob.f90:(.text+0xddb3): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: lincob.f90:(.text+0xddd6): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: lincob.f90:(.text+0xde0e): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: lincob.f90:(.text+0xde2b): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: lincob.f90:(.text+0xde48): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: lincob.f90:(.text+0xde70): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: lincob.f90:(.text+0xdea8): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(lincob.f90.o):lincob.f90:(.text+0xdf2b): more undefined references to `_gfortran_os_error_at' follow
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(newuob.f90.o): in function `__newuob_mod_MOD_newuob':
newuob.f90:(.text+0x167b): undefined reference to `_gfortran_spread'
/usr/bin/ld: newuob.f90:(.text+0x2ebe): undefined reference to `_gfortran_spread'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(uobyqb.f90.o): in function `__uobyqb_mod_MOD_uobyqb':
uobyqb.f90:(.text+0x130e): undefined reference to `_gfortran_spread'
/usr/bin/ld: uobyqb.f90:(.text+0x239c): undefined reference to `_gfortran_spread'
/usr/bin/ld: uobyqb.f90:(.text+0x3cbf): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(initialize.f90.o): in function `__initialize_lincoa_mod_MOD_initxf':
initialize.f90:(.text+0x6bcd): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: initialize.f90:(.text+0x8160): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: initialize.f90:(.text+0x817c): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: initialize.f90:(.text+0x8198): undefined reference to `_gfortran_runtime_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(trustregion.f90.o): in function `__trustregion_lincoa_mod_MOD_trstep':
trustregion.f90:(.text+0x65c8): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(geometry.f90.o): in function `__geometry_lincoa_mod_MOD_geostep':
geometry.f90:(.text+0xc57): undefined reference to `_gfortran_spread'
/usr/bin/ld: geometry.f90:(.text+0x328f): undefined reference to `_gfortran_spread'
/usr/bin/ld: geometry.f90:(.text+0x4fe9): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: geometry.f90:(.text+0x5001): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: geometry.f90:(.text+0x5019): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(geometry.f90.o): in function `__geometry_lincoa_mod_MOD_setdrop_tr':
geometry.f90:(.text+0x5462): undefined reference to `_gfortran_spread'
/usr/bin/ld: geometry.f90:(.text+0x5c0e): undefined reference to `_gfortran_spread'
/usr/bin/ld: geometry.f90:(.text+0x5de3): undefined reference to `_gfortran_spread'
/usr/bin/ld: geometry.f90:(.text+0x5fb7): undefined reference to `_gfortran_spread'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(trustregion.f90.o): in function `__trustregion_newuoa_mod_MOD_circle_fun_trsapp':
trustregion.f90:(.text+0x2f): undefined reference to `sincos'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(trustregion.f90.o): in function `__trustregion_newuoa_mod_MOD_trsapp':
trustregion.f90:(.text+0x2b49): undefined reference to `sincos'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(geometry.f90.o): in function `__geometry_newuoa_mod_MOD_circle_fun_biglag':
geometry.f90:(.text+0x39): undefined reference to `sincos'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(geometry.f90.o): in function `__geometry_newuoa_mod_MOD_circle_fun_bigden':
geometry.f90:(.text+0x163): undefined reference to `_ZGVbN2v_cos'
/usr/bin/ld: geometry.f90:(.text+0x17c): undefined reference to `_ZGVbN2v_cos'
/usr/bin/ld: geometry.f90:(.text+0x1d0): undefined reference to `_ZGVbN2v_sin'
/usr/bin/ld: geometry.f90:(.text+0x1e8): undefined reference to `_ZGVbN2v_sin'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(geometry.f90.o): in function `__geometry_newuoa_mod_MOD_biglag.constprop.0':
geometry.f90:(.text+0x1f2c): undefined reference to `sincos'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(geometry.f90.o): in function `__geometry_newuoa_mod_MOD_geostep':
geometry.f90:(.text+0x562d): undefined reference to `cos'
/usr/bin/ld: geometry.f90:(.text+0x5643): undefined reference to `sin'
/usr/bin/ld: geometry.f90:(.text+0x58c5): undefined reference to `sincos'
/usr/bin/ld: geometry.f90:(.text+0x6b94): undefined reference to `sincos'
/usr/bin/ld: geometry.f90:(.text+0x6d28): undefined reference to `_gfortran_spread'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(geometry.f90.o): in function `__geometry_newuoa_mod_MOD_setdrop_tr':
geometry.f90:(.text+0x7cd5): undefined reference to `_gfortran_spread'
/usr/bin/ld: geometry.f90:(.text+0x84ce): undefined reference to `_gfortran_spread'
/usr/bin/ld: geometry.f90:(.text+0x86a3): undefined reference to `_gfortran_spread'
/usr/bin/ld: geometry.f90:(.text+0x885d): undefined reference to `_gfortran_spread'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(geometry.f90.o):geometry.f90:(.text+0x3df8): more undefined references to `_gfortran_spread' follow
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(trustregion.f90.o): in function `__trustregion_uobyqa_mod_MOD_trstep':
trustregion.f90:(.text+0x42af): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: trustregion.f90:(.text+0x442a): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: /home/zaikunzhang/tmp/prima/install/lib/libprimaf.a(getact.f90.o): in function `__getact_mod_MOD_getact':
getact.f90:(.text+0x43ab): undefined reference to `_gfortran_os_error_at'
/usr/bin/ld: getact.f90:(.text+0x43cd): undefined reference to `_gfortran_os_error_at'
collect2: error: ld returned 1 exit status
gmake[2]: *** [CMakeFiles/cobyla_example.dir/build.make:99: cobyla_example] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/cobyla_example.dir/all] Error 2
gmake: *** [Makefile:136: all] Error 2
bash: ./install/bin/cobyla_example: No such file or directory
```

--- a/README.md
+++ b/README.md
@@ -241,7 +241,7 @@ Below is an illustration with COBYLA.
 cd fortran/examples/cobyla
 cmake -S . -B build -DCMAKE_INSTALL_PREFIX=install -DPRIMA_DIR=$PWD/../../../install/lib/cmake/prima/
 cmake --build build --target install
-./install/bin/cobyla_example
+LD_LIBRARY_PATH=$PWD/../../../install/lib ./install/bin/cobyla_example
 ```
 
 #### C

--- a/c/CMakeLists.txt
+++ b/c/CMakeLists.txt
@@ -15,6 +15,7 @@ set_target_properties(primac PROPERTIES POSITION_INDEPENDENT_CODE ON C_STANDARD 
 
 if (NOT BUILD_SHARED_LIBS)
   target_compile_definitions(primac PUBLIC PRIMAC_STATIC)
+  target_link_libraries (primac INTERFACE ${CMAKE_Fortran_IMPLICIT_LINK_LIBRARIES})
 endif ()
 
 # Export symbols on Windows. See more comments in fortran/CMakeLists.txt.

